File: remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int count = 0;
        for(int i = 1;i < nums.length;i++){
            int one = nums[count];
            int two = nums[i];
            if(one != two){
                count++;
                nums[count] = two;
            }
        }   
        
        return count+1;
    }
}

File: remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.kt
class Solution {
    fun removeDuplicates(nums: IntArray): Int {
     
        var count = 0
        for(i in 1..nums.size-1){
            if(nums[i] > nums[i-1]){
                nums[++count] = nums[i]
            }
        }
        
        return ++count
        
    }
}